ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -379,45 +402,97 @@ StatusDialog {
                     }
 
                     MaxSendButton {
+                        id: maxButton
+
+                        readonly property double maxSafeValue: WalletUtils.calculateMaxSafeSendAmount(
+                                                                   d.maxInputBalance, d.inputSymbol)
+
+                        readonly property double maxSafeCryptoValue: WalletUtils.calculateMaxSafeSendAmount(
+                                                                         d.maxCryptoBalance, d.tokenSymbol)
+
+                        formattedValue: d.currencyStore.formatCurrencyAmount(
```

Comment:
The formatted value always show 0 for fiat


https://github.com/user-attachments/assets/9d9fca13-ea82-4594-af8b-77b817a4066f

Reply:
This is this issue: https://github.com/status-im/status-desktop/issues/15660, present also in master. Not directly related to this PR, I'm going to fix it next.